recipes-multimedia/aamp/aamp_git.bb

```diff
@@ -15,21 +15,27 @@ PV = "0.1.gitr${SRCPV}"
 SRC_URI = "\
     git://github.com/rdkcmf/rdk-aamp.git;protocol=https;branch=stable2 \
     file://0001-rdk-aamp-disable-getsourceid-chech-temporarily-to-sendsyncevent.patch \
+    file://0002-rdk-amp-align-ocdm-drm-adapter-interface.patch \
 "
-SRCREV = "24a4dfa609c7c6b3c18ee595bde0874c9a5ca7c9"
```

Comment:
Can you please add BRANCH variable and make weak assignment for BRANCH and SRCREV?

Reply:
done